src/components/globalConfigurations/GlobalConfiguration.tsx

```diff
@@ -56,11 +57,11 @@ export default function GlobalConfiguration(props) {
     }, [location.pathname])
 
     function getHostURLConfig() {
-        getHostURLConfiguration()
-            .then((response) => {
+        if (props.isSuperAdmin) {
+            getHostURLConfiguration().then((response) => {
                 setIsHostURLConfig(response.result)
             })
-            .catch((error) => {})
+        }
```

Comment:
Removal  of .catch((error) => {}) is appropriate here?

Reply:
yes , it is appropriate as error is not being used here